src/Hacks/antiaim.cpp

```diff
@@ -226,75 +91,77 @@ void AntiAim::CreateMove(CUserCmd* cmd)
         return;
 
-    if (Settings::AntiAim::AutoDisable::noEnemy && localplayer->GetAlive() && !HasViableEnemy())
+    if (localplayer->GetMoveType() == MOVETYPE_LADDER || localplayer->GetMoveType() == MOVETYPE_NOCLIP)
         return;
 
-    QAngle edge_angle = angle;
-    bool edging_head = Settings::AntiAim::HeadEdge::enabled && GetBestHeadAngle(edge_angle);
-
     static bool bSend = true;
     bSend = !bSend;
 
     bool should_clamp = true;
-
     bool needToFlick = false;
+    float tempangle = 0.f;
     static bool lbyBreak = false;
     static float lastCheck;
+    static float nextUpdate = FLT_MAX;
     float vel2D = localplayer->GetVelocity().Length2D();//localplayer->GetAnimState()->verticalVelocity + localplayer->GetAnimState()->horizontalVelocity;
+    CCSGOAnimState* animState = localplayer->GetAnimState();
 
-    if( Settings::AntiAim::LBYBreaker::enabled ){
-
-        if( vel2D >= 0.1f || !(localplayer->GetFlags() & FL_ONGROUND) || localplayer->GetFlags() & FL_FROZEN ){
+    if (Settings::AntiAim::LBYBreaker::enabled)
+    {
+        if (vel2D >= 0.1f || !(localplayer->GetFlags() & FL_ONGROUND) || localplayer->GetFlags() & FL_FROZEN)
+        {
             lbyBreak = false;
             lastCheck = globalVars->curtime;
-        } else {
-            if( !lbyBreak && ( globalVars->curtime - lastCheck ) > 0.22 ){
-                angle.y -= Settings::AntiAim::LBYBreaker::offset;
-                lbyBreak = true;
-                lastCheck = globalVars->curtime;
-                needToFlick = true;
-            } else if( lbyBreak && ( globalVars->curtime - lastCheck ) > 1.1 ){
-                angle.y -= Settings::AntiAim::LBYBreaker::offset;
-                lbyBreak = true;
-                lastCheck = globalVars->curtime;
-                needToFlick = true;
-            }
+            nextUpdate = globalVars->curtime + 0.22;
+        }
+        else if (!lbyBreak && (globalVars->curtime - lastCheck) > 0.22 || lbyBreak && (globalVars->curtime - lastCheck) > 1.1)
+        {
+            tempangle = Settings::AntiAim::LBYBreaker::offset;
+            lbyBreak = true;
+            lastCheck = globalVars->curtime;
+            nextUpdate = globalVars->curtime + 1.1;
+            needToFlick = true;
         }
     }
 
-    if (Settings::AntiAim::Yaw::enabled && !needToFlick)
-    {
-        DoAntiAimY(localplayer, angle, bSend);
+    if ((nextUpdate - globalVars->interval_per_tick) >= globalVars->curtime && nextUpdate <= globalVars->curtime)
+        CreateMove::sendPacket = false;
 
-        CreateMove::sendPacket = bSend;
-        if (Settings::AntiAim::HeadEdge::enabled && edging_head && !bSend)
-            angle.y = edge_angle.y;
-    }
+    static bool directionSwitch = false;
+
+    if (inputSystem->IsButtonDown(KEY_LEFT))
+		directionSwitch = true;
+	else if (inputSystem->IsButtonDown(KEY_RIGHT))
+		directionSwitch = false;
 
-    if (!ValveDSCheck::forceUT && (*csGameRules) && (*csGameRules)->IsValveDS())
```

Comment:
Why would you remove DS check?

Reply:
I removed the feature to change pitch. I figured it would be useless.